modules/gdscript/language_server/godot_lsp.h

```diff
@@ -112,6 +120,27 @@ struct Range {
 	 */
 	Position end;
 
+	_FORCE_INLINE_ bool operator==(const Range &p_other) const {
+		return start == p_other.start && end == p_other.end;
+	}
+
+	bool contains(const Position p_pos) const {
```

Comment:
Any reason this isn't passed by reference here?

Reply:
Not really. But neither any reason to be passed by reference: `Position` is very small (and copying is most likely faster -- probably what the compiler does anyway?)

Reply:
We do it elsewhere for it, so uniformity, and it doesn't cost anything to do so